### PR TITLE
feat: add parents_concatenate taxon name and id endpoint

### PR DESCRIPTION
**Pull request by MagicTearsAsunder**
Issue: #250

### CODE REVIEW

File: taxonomy/app/schemas/taxon.py [hunk 1]
```diff
@@ -39,3 +39,18 @@ class TaxonResponseSchema(TaxonInputSchema):
...
+    taxon_name: str = Field(..., example="taxon_name")
+    parent_ids_concat: Optional[str] = Field(
+        ..., example="parent_id_1.parent_id_2"
+    )
+    parent_names_concat: Optional[str] = Field(
+        ..., example="parent_name_1.parent_name_2"
+    )
+
+    class Config:
+        allow_population_by_field_name = True
```
Comment (spiridonovfed) [inline taxonomy/app/schemas/taxon.py:55]: You don't use aliases for model fields. So there's no point of using allow_population_by_field_name = True

File: taxonomy/app/schemas/taxon.py [hunk 2]
```diff
@@ -39,3 +39,18 @@ class TaxonResponseSchema(TaxonInputSchema):
...
+        orm_mode = True
```
Comment (spiridonovfed) [inline taxonomy/app/schemas/taxon.py:56]: Do you this model with ORM?
If not - there's no point of using this attribute

File: taxonomy/app/schemas/taxon.py (hunk 1 above)
Comment (MagicTearsAsunder) [reply taxonomy/app/schemas/taxon.py]: agree, removed both

File: taxonomy/app/schemas/taxon.py (hunk 2 above)
Comment (MagicTearsAsunder) [reply taxonomy/app/schemas/taxon.py]: agree, removed both

**Review by spiridonovfed — APPROVED** [review]

**Pull request merged**